### PR TITLE
Merge development into main

### PR DESCRIPTION
**Pull request by axtloss**
Changes:
- migrate from blueprint to xml/ui
- Adds manual partitioning
- Fixes wording/phrasing of text elements

### CODE REVIEW

**Comment by mchalunderscore** [comment]
Please name the PR and provide at least a minimal changelog in the description going forward

**Comment by axtloss** [comment]
> Please name the PR and provide at least a minimal changelog in the description going forward

Alright, going to do that.

File: src/functions/misc_screen.py [hunk 1]
```diff
@@ -28,12 +28,14 @@ class MiscScreen(Adw.Bin):
     ipv_switch = Gtk.Template.Child()
     timeshift_switch = Gtk.Template.Child()
     zramd_switch = Gtk.Template.Child()
+    #theme_switch = Gtk.Template.Child()
```
Comment (mchalunderscore) [inline src/functions/misc_screen.py]: What's this? I thought if we would do any potential theming related options they would be in Moonstone?

Comment (axtloss) [reply src/functions/misc_screen.py]: This is just leftover code from when we wanted to have it in jade, I'll remove it.

Comment (mchalunderscore) [reply src/functions/misc_screen.py]: Sounds good, it seems to be in `summary_screen.py` too

**Review by mchalunderscore — APPROVED** [review]

**Pull request merged**